src/deepsparse/benchmark/benchmark_model.py

```diff
@@ -379,7 +388,6 @@ def benchmark_model(
 
     export_dict = {
         "engine": str(model),
```

Comment:
Could you do most of the parsing from below here instead?
```suggestion
        **yaml.load("engine: " + str(model).replace("\t", "").replace(":", "", 1)),
```

Reply:
Alternatively you could use the `model._properties_dict()` (https://github.com/neuralmagic/deepsparse/blob/main/src/deepsparse/engine.py#L545) directly to get the dictionary

Reply:
+1 for the `_properties_dict` as the output structure of `str(model)` is subject to change

Reply:
`_properties_dict` will populate the metadata from a result from a func, right now its populating straight from the input arg.

str(model) has engine key with messy output which is the reason for this pr.